packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -0,0 +1,127 @@
+import { IModelsClient } from "@itwin/imodels-client-authoring";
+import { TestBrowserAuthorizationClient } from "@itwin/oidc-signin-tool";
+import { Reporter } from "@itwin/perf-tools";
+import * as os from "os";
+
+async function sleep(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+const testCasesMap = new Map<string, any>();
+testCasesMap.set("identity transform", require("./identity-transformer"));
+// testCasesMap.set("prepare fork transform", require("./prepare-fork"));
+
+const loggerCategory = "Transformer Performance Tests Identity";
+const outputDir = path.join(__dirname, ".output");
+
+const setupTestData = async () => {
+  const logLevel = +process.env.LOG_LEVEL!;
+  if (LogLevel[logLevel] !== undefined) {
+    Logger.initializeToConsole();
+    Logger.setLevelDefault(LogLevel.Error);
+    Logger.setLevel(loggerCategory, LogLevel.Info);
+    Logger.setLevel(TransformerLoggerCategory.IModelExporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelImporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelTransformer, logLevel);
+  }
+  let usrEmail;
+  let usrPass;
+  if(process.env.V2_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V2_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V2_CHECKPOINT_USER_PASSWORD;
+  } else if(process.env.V1_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V1_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V1_CHECKPOINT_USER_PASSWORD;
+  } else {
+    usrEmail = process.env.IMODEL_USER_NAME;
+    usrPass = process.env.IMODEL_USER_PASSWORD;
+  }
+
+  assert(usrEmail, "user name was not configured");
+  assert(usrPass, "user password was not configured");
+  const user = {
+    email: usrEmail,
+    password: usrPass,
+  };
+
+  assert(process.env.OIDC_CLIENT_ID, "");
+  assert(process.env.OIDC_REDIRECT, "");
+  const authClient = process.env.CI === "1"
+    ? new TestBrowserAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: "itwins:read imodels:read imodels:modify",
+      authority: "https://qa-ims.bentley.com",
+    }, user)
+    : new NodeCliAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: "imodelaccess:read storage:modify realitydata:read imodels:read library:read imodels:modify realitydata:modify savedviews:read storage:read library:modify itwinjs savedviews:modify",
+    });
+
+  await authClient.signIn();
+
+  const hostConfig  = new IModelHostConfiguration();
+  hostConfig.authorizationClient = authClient;
+  const hubClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX}api.bentley.com/imodels` } });
+  hostConfig.hubAccess = new BackendIModelsAccess(hubClient);
+  await IModelHost.startup(hostConfig);
+
+  const arrImodels = [];
+  for await (const iModel of getTestIModels()) {
```

Comment:
this sort of defeats the purpose of that iterator? Might as well rewrite getTestIModels?

Reply:
I mentioned in another comment that mocha doesn't like 4 loops. Don't know how to rewrite this without a for loop, so that it could be used directly inside the mocha test instead of needing it as a middle man